src/core/Flora/Import/Categories.hs

```diff
@@ -1,22 +1,22 @@
 module Flora.Import.Categories where
 
 import Control.Monad.IO.Class
+import Data.Text (Text)
 import Data.Text.IO qualified as T
 import Effectful
 import Effectful.PostgreSQL.Transact.Effect
 
-import Flora.Import.Categories.Tuning as Tuning
 import Flora.Model.Category.Types (Category, mkCategory, mkCategoryId)
 import Flora.Model.Category.Update (insertCategory)
+import Flora.Normalise
 
 importCategories :: (DB :> es, IOE :> es) => Eff es ()
 importCategories = do
-  liftIO $ T.putStrLn "Sourcing categories from Datalog"
-  canonicalCategories <- liftIO Tuning.sourceCategories
-  categories <- mapM fromCanonical canonicalCategories
+  liftIO $ T.putStrLn "Sourcing categories"
+  categories <- mapM fromCanonical floraCategories
```

Comment:
The litmus test for the usage of `floraCategories` is `is it replacing `sourceCategories`. If not, then you don't want `floraCategories`.